utils/scripts/find_duplicate_read_sets/README.md

```diff
@@ -0,0 +1,50 @@
+# AWS HealthOmics Sequence Store Duplicate Read Set Finder script
+
+This is provided AS-IS and is intended to aid in identifying read sets that are duplicate within and across sequence stores in the same region. This is a starting point, feel free to customize as needed to fit your specific requirements.
+
+## find_duplicate_read_sets.py
+
+This script was written to help in the identification of duplicate read sets stored in [AWS HealthOmics sequence stores](https://docs.aws.amazon.com/omics/latest/dev/what-is-service.html). This script relies on [HealthOmics ETags](https://docs.aws.amazon.com/omics/latest/dev/etags-and-provenance.html) which is a hash of the ingested content in a sequence store. 
+
+To run this script, simply identify the sequence store IDs you want to search across in a region, specify the input parameters, and you receive a CSV with the read set sources that share an ETag. 
+
+This script makes a few assumptions:
+1. The boto3 configured by default in your python account or the passed in profile has access to list read sets in the HealthOmics sequence stores identified. 
```

Comment:
reword:
1. Python 3.10 or above is installed along with the AWS SDK for Python (`boto3`)
2. AWS CLI v2 is installed and configured with a profile with read access to AWS HealthOmics sequence stores

similarly, if there are other Python package requirements, I would call them out as well. Even better would be to include a `requirements.txt` file that users can use `pip install` against.

Reply:
updated and included a requirements.txt